src/vaadin-checkbox.html

```diff
@@ -84,7 +84,7 @@
        * Part name         | Description
        * ------------------|----------------
        * `checkbox`        | The checkbox element
-       * `label`           | The label content element
+       * `label`           | The label generated from the light DOM content
```

Comment:
What is generated? Looking at the template, `part=label` is just a [wrapper for slotted content](https://github.com/vaadin/vaadin-checkbox/blob/master/src/vaadin-checkbox.html#L63-L65).